src/main/java/lk/gov/health/phsp/bean/ClientController.java

```diff
@@ -1575,14 +1604,14 @@ public void clearSearchById() {
     }
 
     public List<Client> listPatientsByNic(String phn) {
-        String j = "select c from Client c where c.retired=false and upper(c.person.nic)=:q order by c.phn";
+        String j = "select c from Client c where c.retired=false and c.reservedClient<>true and upper(c.person.nic)=:q order by c.phn";
         Map m = new HashMap();
         m.put("q", phn.trim().toUpperCase());
         return getFacade().findByJpql(j, m);
     }
 
     public List<Client> listPatientsByPhone(String phn) {
-        String j = "select c from Client c where c.retired=false and (upper(c.person.phone1)=:q or upper(c.person.phone2)=:q) order by c.phn";
+        String j = "select c from Client c where c.retired=false and c.reservedClient<>true and (upper(c.person.phone1)=:q or upper(c.person.phone2)=:q) order by c.phn";
```

Comment:
Need to put true as a parameter

Reply:
I thought in listPatientsByNic --> definitely should eliminate items that those values do not exist. Here these records does not contain nic value and phone no. So my thinking was we can directly remove those from the result list

Reply:
THis is not needed. Just practice. 
What I suggested is 

c.reservedClient<>:rs **instead of** c.reservedClient<>true
add m.put("rs",true)